AdminUi/src/AdminUi/ClientApp/src/app/services/quotas-service/quotas.service.ts

```diff
@@ -60,6 +60,7 @@ export interface Quota {
     metric: Metric;
     max: number;
     period: string;
+    tierDisabled: boolean;
```

Comment:
Why is this property called `tierDisabled`? Shouldn't it just be called `disabled`? The UI doesn't care if it is a tier quota or an individual quota